Details screen

henriquehsv opened the pull request.

Commit:
Addind a stub for a details screen

Commit:
Communicating details screen and main screen

Commit:
Added more elements to details screen

sergiosilvajr commented on app/src/main/java/br/com/example/android/popularmovies/view/MovieDetailsActivity.java:
```diff
@@ -0,0 +1,32 @@
+
+        Toolbar toolbar = (Toolbar) findViewById(R.id.toolbar);
+        setSupportActionBar(toolbar);
+        getSupportActionBar().setDisplayHomeAsUpEnabled(true);
```
Could we take SupportActionBar reference to check if it is null before using
its methods? (Just to avoid null pointers)

henriquehsv replied:
I think there is a better way to do that using MVVM. Will update the code.

henriquehsv replied:
It doesn't make sense testing the return of getSupportActionBar, since we just used setSupportAction bar.

sergiosilvajr commented on app/src/main/res/layout/activity_movie_details.xml:
```diff
@@ -0,0 +1,130 @@
+                android:layout_alignParentBottom="true"
+                android:layout_below="@id/moviePoster"
+                android:background="@color/colorPrimary"
+                android:padding="8dp"
```
Could we put this value on dimen file?

sergiosilvajr commented:
👍  with minors comments

Commit:
Split movie details into one include

Created another ViewModel to describe how the movie layout should behave
depending on the screen.

sergiosilvajr commented:
👍
